README.md

```diff
@@ -6,7 +6,41 @@
 [![PypiVersion](https://img.shields.io/pypi/v/metalearners.svg?logo=pypi&logoColor=white)](https://pypi.org/project/metalearners)
 [![codecov.io](https://codecov.io/github/QuantCo/metalearners/coverage.svg?branch=main)](https://codecov.io/github/QuantCo/metalearners?branch=main)
 
-MetaLearners for CATE estimation
+MetaLearners for Conditional Average Treatment Effect (CATE) estimation
+
+The library focuses on providing
+
+- Methodologically sound cross-fitting
+- Convenient access to and reuse of base models
+- Consistent APIs across Metalearners
+- Support for multi-class classification and more than binary treatment variants
```

Comment:
Nit, multi-class only works for some metalearners, not sure if we should put it as a focus but feel free to ignore

Reply:
Oh yes; great catch!

Reply:
[ee6c10a](https://github.com/Quantco/metalearners/pull/44/commits/ee6c10ae072dd6219561ec797d95cf4176b5b730)